app/assets/javascripts/app/utils/pluralize.js

```diff
@@ -0,0 +1,6 @@
+export default (word, count) => {
+  const I18n = window.LoginGov.I18n;
+  const tString = I18n[word] || word;
+
+  return `${tString}${count !== 1 ? 's' : ''}`;
```

Comment:
For the time being, since we are only planning on English/Spanish which have [similar](http://unicode.org/repos/cldr-tmp/trunk/diff/supplemental/language_plural_rules.html#en) [pluralization](http://unicode.org/repos/cldr-tmp/trunk/diff/supplemental/language_plural_rules.html#es), we can probably hardcode the `plural = count == 1 ? false : true` kind of thing, but eventually we may want a JS library that uses the unicode CLDR to handle multiple locales (I think i18n.rb already includes this kind of logic)